services/comparison/types.py

```diff
@@ -17,7 +17,19 @@ class FullCommit(object):
 @dataclass
 class Comparison(object):
     head: FullCommit
+
+    # To see how a patch changes project coverage, we compare the branch head's
+    # report against the base's report, or if the base isn't in our database,
+    # the next-oldest commit that is. Be aware that this base commit may not be
+    # the true base that, for example, a PR is based on.
     base: FullCommit
```

Comment:
I wonder if using `project_coverage_base` and `patch_coverage_base` would be better names from an understanding perspective. 

Although the comments do a good job of explaining the difference.

Reply:
oh you're right those are much better names. i tried renaming `base` to `adjusted_base` and it was a very messy rename so i backed out since merging this was a priority, but i assigned myself a GH issue to rename the fields with your suggested names when i get a spare minute https://github.com/codecov/worker/issues/203